caldav/server_test.go

```diff
@@ -2,7 +2,10 @@ package caldav
 
 import (
 	"context"
+	"encoding/xml"
 	"fmt"
+	"github.com/emersion/go-webdav/internal"
+	"github.com/stretchr/testify/assert"
```

Comment:
I'd prefer to not add the testify dependency.

Reply:
Testify allows to write a bit more compact test code. 
Which improves readability.

Reply:
It's a whole new library to learn. I prefer to write tests like the Go standard library does.